src/test/java/ru/training/at/hw2/ex1/JdiPageSeleniumFirstTest.java

```diff
@@ -0,0 +1,194 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.Assert.assertEquals;
+
+public class JdiPageSeleniumFirstTest {
+    WebDriver webDriver;
+    final String testPageUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+    final String userName = "Roman";
+    final String password = "Jdi1234";
+
+    @BeforeMethod(alwaysRun = true)
+    public void driverSetup() {
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Drivers\\chromedriver_win32\\chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        /*webDriver.manage().timeouts()
+                .implicitlyWait(10, TimeUnit.SECONDS);*/
+    }
+
+    @Test
+    public void jdiSeleniumPageFirstTest() {
+
+        // 1. Assert that test site page is loaded
+        webDriver.manage().timeouts().pageLoadTimeout(10, TimeUnit.SECONDS);
+        webDriver.navigate().to(testPageUrl);
+        SoftAssert softAssert = new SoftAssert();
+        WebDriverWait webDriverWait = new WebDriverWait(webDriver, 10);
+        softAssert.assertEquals(testPageUrl, webDriver.getCurrentUrl());
+
+        //2. Assert that browser title is "Home page"
+        softAssert.assertEquals(webDriver.getTitle(), "Home Page");
+
+        //3. Assert that user is logged
+        WebElement loginDropdown = webDriverWait
+                .until(webDriver -> webDriver.findElement(By.xpath("//a[@href='#']")));
```

Comment:
I suppose that here could be used more readable locator for the dropdown of user

Reply:
changed the locator